blog/_posts/2017-11-01-zinc-blog-1.0.md

```diff
@@ -0,0 +1,280 @@
+* Add JMH benchmarks to reason about the performance impact of incremental compiles.
+* Improve infrastructure to attract contributors:
+  * Speed up CI by 7x to have a fast turnaround time,
+  * Add more tests to the project,
+  * Add more documentation to the APIs, a README and a CONTRIBUTING guide; and,
+  * Add similar high-quality improvements to the build and scripted.
+  
+Find the [list of PRs here](https://github.com/sbt/zinc/pulls?utf8=%E2%9C%93&q=author%3Ajvican%20is%3Ap://github.com/sbt/zinc/pulls?utf8=%E2%9C%93&q=author%3Ajvican%20is%3Apr).
+  
+### External contributions
+
+Zinc 1.0.0 also has had significant contributions from the Scala open-source
+community, and I thank everyone involved to make it possible! The contributors
+since our involvement in the project have been [Stu Hood][], [Peiyu Wang][],
+[Krzysztof Romanowski][], [Wieslaw Popielarski][], [Peter Pan][], [Gregor
+Heine][], [Łukasz Indykiewicz][], [Allan Timothy Leong][], [Guillaume
+Martres][], [Wojciech Langiewicz][], [Krzysztof Borowski][], [Wojtek Pitula][],
+[knirski][], [Thierry Treyer][], [Grzegorz Kossakowski][], [Dale Wijnand][] and
+[Eugene Yokota][].
+
+[Martin Duhem] and I are happy that we met some of these contributors during our
+**[Scala Center sprees](https://github.com/scalacenter/sprees)** all over Europe
+and that they dared to submit pull requests to Zinc; while contributing to the
+project may at first sound daunting, you didn't give up and at the end your work
+had a clear impact on Scala developers. Congratulations!
+
+### A preview of Zinc 1.0: an example of a 7x speedup
+  
+How big are the improvements that Zinc 1.x brings to the table and how do they
+impact your day-to-day developer workflow?
+  
+To answer this question, we're going to use both Zinc 0.13.x and 1.x to compile
+a big real-world project: [Scalatest](http://www.scalatest.org), with
+**405.084** lines of Scala code without counting comments and blank lines.
+  
+We illustrate our example with a common and relatively uninstrusive operation in
+every codebase: a method addition. In our case, we add a new method to a widely
+dependent class like `AndHaveWord` (in `Matcher.scala`) after we've correctly
+warmed up the compiler (note that it's not fully hot) and compiled the project.
+How fast can Zinc 0.13.x recompile the new method?
+  
+[![scastie](/resources/img/blog/zinc-0.13-scalatest.gif)](/resources/img/blog/zinc-0.13-scalatest.gif)
+  
+The incremental compilation in **Zinc 0.13.x yields a compilation time of 21
+seconds**.  Let's try it with Zinc 1.0.0.
+  
+[![scastie](/resources/img/blog/zinc-1.0-scalatest.gif)](/resources/img/blog/zinc-1.0-scalatest.gif)
+  
+**Zinc 1.0.0** terminates **compilation in only 3 seconds**. For this particular
+example, the class-based dependency in the new version does 7x better
+incremental compilation than the source-based old version.
+  
+This experiment is subject to diverse results depending on the scenarios, but
```

Comment:
"This experiment is subject to diverse results depending on the scenarios"

Sentence can be improved. Subject to what scenarios? Too vague...